R/uvhydrograph-styles.R

```diff
@@ -6,6 +6,16 @@ getUvStyle <- function(data, info, correctionLabels, plotName, dataSides, dataLi
   comp_lbl <- info$comp_UV_lbl
   comp_type <- info$comp_UV_type
   legend.name <- data[[1]]$legend.name
+  corrArrowPositions <- list()
+  plotEndDate <- tail(info$plotDates,1) + hours(23) + minutes(45)
+  plotStartDate <- info$plotDates[1]
+
+  #Make the correction label lines connect to the outside of the bounding box and not to the center of the label
+  if(!isEmptyOrBlank(correctionLabels)){
+    corrArrowPositions <- correctionLabels %>% as.data.frame() %>% select(x, xorigin, r, y) %>%
```

Comment:
The `cumsum` functions using `xpos` is what changes the value from POSIXct to numeric. I think we can just convert back, but you'll need the timezone. So, just add `timezone <- getReportMetadata(data,'timezone')` right below the `isInverted` line in `parseUVSupplemental` and then you can use the timezone inside of `parseLabelSpacing`  by saying `info$timezone`.